### PR TITLE
Update fms-hf-tuning to latest released image

### PR DESCRIPTION
**Pull request by abhijeet-dhumal**
Reference PR : https://github.com/red-hat-data-services/rhoai-additional-images/pull/20

### CODE REVIEW

File: ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource [hunk 1]
```diff
@@ -12,7 +12,7 @@ ${CODEFLARE-SDK_REPO_URL}                %{CODEFLARE-SDK_REPO_URL=https://github
 ${DISTRIBUTED_WORKLOADS_RELEASE_ASSETS}  https://github.com/opendatahub-io/distributed-workloads/releases/latest/download
 ${RAY_IMAGE_3.11}                        quay.io/modh/ray@sha256:db667df1bc437a7b0965e8031e905d3ab04b86390d764d120e05ea5a5c18d1b4
 ${RAY_IMAGE_3.9}                         quay.io/modh/ray@sha256:0d715f92570a2997381b7cafc0e224cfa25323f18b9545acfd23bc2b71576d06
-${FMS_HF_TUNING_IMAGE}                   quay.io/modh/fms-hf-tuning@sha256:73bcd66500b8637a9db1339f64c3217212ef74700a22a790f78e9a1f26b8b71a
+${FMS_HF_TUNING_IMAGE}                   quay.io/modh/fms-hf-tuning@sha256:ae61e3f96ab3e6ddaa33978fc103215d500e1f2a12d5c53d634257e9e058753a
```
Comment (ChughShilpa) [inline ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource:15]: This is not the latest release image.
 I know same image is present in https://github.com/red-hat-data-services/rhoai-disconnected-install-helper/blob/main/rhoai-2.16.md as it is not updated. It just copied the same image from https://github.com/red-hat-data-services/rhoai-disconnected-install-helper/blob/main/rhoai-2.15.md

Comment (ChughShilpa) [reply ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource:15]: Also we prefer to use specific version as mentioned [here](https://github.com/red-hat-data-services/rhoai-additional-images/pull/14#issuecomment-2404946581)

Comment (abhijeet-dhumal) [reply ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource:15]: Ahh, I see
Actually i saw [this](https://github.com/red-hat-data-services/rhoai-additional-images/pull/20) PR raised a Week ago which had same image updated.. :eyes:

Comment (abhijeet-dhumal) [reply ods_ci/tests/Resources/Page/DistributedWorkloads/DistributedWorkloads.resource:15]: Okay, Just now i started with disconnected verification with 2.16 Rc where as you mentioned it has the same updated fms-image which was updated during 2.15 in the disconnected-imageset 2.16-CR1
So to run tests using the updated image with master branch, i have created this branch to run tests with!
Let me close this PR then!!

Thank you @ChughShilpa for pointing this out, i missed those details :raised_hands:

**Pull request closed without merging**